components/css/partials/_quote.scss

```diff
@@ -14,7 +14,7 @@
         background: $color-campusGold;
     }
 
-    &-quotemark {
+    &-mark {
         padding-left: 4rem;
 
         &::before {
```

Comment:
So I just realized that screen readers will read "left double quotation mark" separately in front of the quote title/content for this modifier. I'm not quite sure how to handle this. We could replace the content with the FontAwesome icon, https://fontawesome.com/icons/quote-left?style=solid , but that still has the issue of the screen reader coming upon a separate element that it can't read.

We could use an SVG background image and make the the content property and empty string http://jsfiddle.net/6WAtQ/ . We'd have to set the height and width equal to rems instead of the font-size as well. That way, the screen reader would have nothing to read in the first place.

Thoughts?

Reply:
I decided to cook up a couple previews of what using SVG inline would look like:

## Outline of the Archivo Narrow Curly Quote:
https://jsfiddle.net/jvenabl/hj8p26zv/23/
**Pros:**
- Easier to position

**Cons:**
- Can't change quote font as easily since it's an outline

## Actual text in the SVG:
https://jsfiddle.net/jvenabl/zrf1jffj/39/
**Pros:**
- Can easily change the quote font by just changing the font-family.

**Cons:**
- If you update the font, you need to mess with text element's position and possibly font size (it's kind of a pain).

Reply:
What about a "speech" media query?

```
@media speech {
    .quote-mark::before { {
        display: none;
        visibility: hidden;
    }
}
```

Reply:
No go. Doesnt work.

Reply:
Just for reference I played around with a few other ideas with no luck: `speak: none;` and `speak-as: no-punctuation;`

I'd opt for option one since I don't think we're going to be changing it often, and even if we did, I don't think the process would be too hard.

Reply:
I've implemented option 1. It will be reflected in the next push.